Factorialize/C++/Factorialize.cpp

```diff
@@ -0,0 +1,11 @@
+int factorial(int num){
+	// Default cases - Both 0! & 1! will return 1
+	if(num == 0 || num == 1){
```

Comment:
This `if` statement is not required, since total is initialized to 1 (line 6) and the for loop is for `i>1`. If the number provided is 0 or 1, the `for` loop will not evaluate anything and the total will be returned normally.